frontend/src/components/auth/auth.module.scss

```diff
@@ -0,0 +1,110 @@
+.formWrapper {
+  display: flex;
+  flex-direction: column;
+  width: 600px;
+  margin-right: 60px;
+  padding: 40px;
+  color: white;
+  background-color: var(--background-gray-400);
+  transform: translateY(-50%);
+}
+
+.main {
+  display: flex;
+  justify-content: right;
+  align-items: center;
+  height: 100vh;
+}
+
+.formHeader {
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+}
+
+.formHeader h2 {
+  font-weight: 600;
+  font-size: 30px;
+}
+
+.formHeader a {
+  color: var(--support-info-blue-100);
+  text-decoration: none;
+}
+
+.formHeader h2,
+p {
+  margin: 0;
+}
+
+.formContent {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 24px 0;
+}
+
+.buttonWrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.background {
```

Comment:
It is possible to do so, but it will reveal the issues with overflow. I decided to position background as a separate component, so that there's no issues between background and form. Do you think there's a better way to do it?

Reply:
oh ok, I see, let's leave it as it is